Update and clarify NGINX One RBAC docs (#265)
nginx/documentation: content/nginx-one/rbac/rbac-api.md

docs: DOCS-000
---

> Beyond the [Default roles]({{< relref "/nginx-one/rbac/roles.md" >}}) for NGINX One Console access, there may be cases where creating [custom roles](https://docs.cloud.f5.com/docs-v2/administration/how-tos/user-mgmt/roles#custom-roles) with more precisely defined access is desired. For this use-case, we include a list of API Groups that can be used to specify permissions for custom roles with more granular access controls to NGINX One Console APIs.

Reviewer: API Groups is capitalized here but lowercase in the heading in Ln 14.

Is this intended?

(I think it should be lowercase for both)

> Beyond the [Default roles]({{< relref "/nginx-one/rbac/roles.md" >}}) for NGINX One Console access, there may be cases where creating [custom roles](https://docs.cloud.f5.com/docs-v2/administration/how-tos/user-mgmt/roles#custom-roles) with more precisely defined access is desired. For this use-case, we include a list of API groups that can be used to specify permissions for custom roles with more granular access controls to NGINX One Console APIs.

Reviewer 2: ```suggestion
Beyond the [Default roles]({{< relref "/nginx-one/rbac/roles.md" >}}) for NGINX One Console access, you can create [custom roles](https://docs.cloud.f5.com/docs-v2/administration/how-tos/user-mgmt/roles#custom-roles) with more precisely defined access. For this use-case, we include a list of API groups that you can use to specify permissions for custom roles with more granular access controls to NGINX One Console APIs.
```

> A custom roles can be assigned to users or service accounts for specific namespaces, offering a way to maintain principle of least privilege across your tenant.

Author: fix:

either `A custom role` or `Custom roles`

> Custom roles can be assigned to users or service accounts, and associated with specific namespaces, offering a way to maintain principle of least privilege across your tenant.

Reviewer 3: ```suggestion
Custom roles can be assigned to users or service accounts, and associated with specific namespaces, facilitating the principle of least privilege across your tenant.
```

## F5 API groups for NGINX One

The following table lists the available API Groups that you can use to construct a Role. These are narrowly scoped API Groups that align with all the features and functionality within the NGINX One Console. These groups can help you create custom roles tailored to your specific needs.

> {{< note >}}If you create custom roles using these API Groups, users may not have access to all capabilities of the UI Console.{{< /note >}}

Reviewer: ```suggestion
{{< note >}}If you create custom roles using these API Groups, users may not have access to all capabilities of the user interface console.{{< /note >}}
```

See my other comment about groups being capitalized too

Reviewer: and I think console should be lowercase unless you write the complete product name, NGINX One Console

Author: changed to `browser web portal`

I feel like if UI isn't clear to a reader, then user interface is worse, not better. Hopefully this is more clear about what we are referring to.

let me know if you have further thoughts.
